borgmatic/logger.py

```diff
@@ -41,6 +41,10 @@ def should_do_markup(no_color, configs):
     if any(config.get('output', {}).get('color') is False for config in configs.values()):
         return False
 
+    no_color_env = os.environ.get('NO_COLOR', None)
+    if no_color_env is not None and to_bool(no_color_env):
```

Comment:
I don't think this is quite right if I'm reading the [specification](https://no-color.org/) correctly. It sounds like `NO_COLOR` is considered "set" (true) whenever it has *any* value—not just when it has a boolean value that evaluates to true. From the spec:

> Command-line software which adds ANSI color to its output by default should check for a NO_COLOR environment variable that, when present and not an empty string (regardless of its value), prevents the addition of ANSI color.

Reply:
Oh yeah you're right, I'll fix it.